soh/soh/GameMenuBar.cpp

```diff
@@ -1084,6 +1084,8 @@ namespace GameMenuBar {
                     UIWidgets::PaddedEnhancementCheckbox("Remove power crouch stab", "gCrouchStabFix", true, false);
                     UIWidgets::Tooltip("Make crouch stabbing always do the same damage as a regular slash");
                 }
+                UIWidgets::PaddedEnhancementCheckbox("Fix Gerudo Warrior's clothing colors", "gGerudoWarriorClothingFix", true, false);
+                UIWidgets::Tooltip("Fixes the colors of the Gerudo Warrior's clothes when changing Link's clothes or using bombs in front of her");
```

Comment:
Hmm, I'm not sure there's a better way to describe it...

Reply:
In this case it's also more an "Unfix" than a "Fix" since it was a big that has been fixed and you're unfixing it

I would think a better wording necessary for this case

Reply:
Yes, but checking the option will "fix" it, it is fixed in the MQ versions...

Reply:
Sounds good to me then